src/lib/services/types/tx.ts

```diff
@@ -185,6 +187,19 @@ export const zTxsByAddressResponseLcd = z
   }));
 export type TxsByAddressResponseLcd = z.infer<typeof zTxsByAddressResponseLcd>;
 
+export const zTxsByAddressResponseSequencer = z
+  .object({
+    txs: z.array(zTxsResponseItemFromLcd),
+    pagination: zPagination,
+  })
+  .transform((val) => ({
+    items: val.txs,
+    pagination: val.pagination,
+  }));
+export type TxsByAddressResponseSequencer = z.infer<
+  typeof zTxsByAddressResponseSequencer
+>;
```

Comment:
change `total` to `zTxsByAddressResponseLcd` to `pagination` as we discussed to use cursor-based pagination instead, then just reuse `zTxsByAddressResponseLcd` in `getTxsByAccountAddressSequencer`